### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by Pritam0077**
I have added the MIT license

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Kendall Pinto
```
Comment (KendallDoesCoding) [inline LICENSE:3]:
```suggestion
Copyright (c) 2022 KendallDoesCoding
```

**Review by KendallDoesCoding — COMMENTED** [review]
update copyright name.

**Review by KendallDoesCoding — APPROVED** [review]
LGTM ✔️

**Comment by KendallDoesCoding** [comment]
I'm merging this now, but I need to get #4 done asap.

**Comment by KendallDoesCoding** [comment]
Thanks a lot for your contribution @Pritam0077

**Comment by Pritam0077** [comment]
Your welcome

**Comment by KendallDoesCoding** [comment]
btw, i need some help, just dropped you an email on the email listed on
your profile README.

*Thanks,*
*Kendall Does Coding*

**Pull request merged**